app/experimenter/nimbus-ui/src/components/App/index.test.tsx

```diff
@@ -22,16 +38,50 @@ describe("App", () => {
     expect(screen.getByTestId("PageNew")).toBeInTheDocument();
   });
 
-  const Subject = ({ basepath = "/", path = "/" }) => {
-    let source = createMemorySource(path);
-    let history = createHistory(source);
-
-    return (
-      <LocationProvider history={history}>
-        <App {...{ basepath }} />
-      </LocationProvider>
+  describe(":slug routes", () => {
+    const Root = (props: any) => <>{props.children}</>;
+    const Routes = () => (
```

Comment:
So yeah, this is gross and I don't like explicitly sticking this here instead of just pulling in `App` and mocking the imports like we are for the two tests above this in the file, but I'm having trouble rendering `<App />` here with `:slug` being properly grabbed in `AppLayoutWithSidebar`. Looking for a better solution.